doc/html/Command Line Usage.html

```diff
@@ -74,20 +74,20 @@ <h1>Command Line Usage</h1>
 </tr>
 <tr>
 <td><em>/beep</em> or <em>/b</em></td>
-<td>Beep after a volume has been successfully mounted or dismounted.</td>
+<td>Beep after a volume has been successfully mounted or unmounted.</td>
 </tr>
 <tr>
 <td><em>/auto</em> or <em>/a</em></td>
 <td>If no parameter is specified, automatically mount the volume. If devices is specified as the parameter (e.g., /a devices), auto-mount all currently accessible device/partition-hosted VeraCrypt volumes. If favorites is specified as the parameter, auto-mount
  favorite volumes. Note that /auto is implicit if /quit and /volume are specified. If you need to prevent the application window from appearing, use /quit.</td>
 </tr>
 <tr>
-<td><em>/dismount</em> or <em>/d</em></td>
-<td>Dismount volume specified by drive letter (e.g., /d x). When no drive letter is specified, dismounts all currently mounted VeraCrypt volumes.</td>
+<td><em>/unmount</em> or <em>/d</em></td>
+<td>Unmount volume specified by drive letter (e.g., /d x). When no drive letter is specified, unmounts all currently mounted VeraCrypt volumes.</td>
```

Comment:
The documentation entry should be corrected: /unmount comes with shortened version /u and deprecated /dismount is still supported with shortened /d.
I proposed to keep an entry for /dismount that indicate that it is deprecated and similar to /unmount and then add new entry for /unmount with correct shortened version.
For example:

```
<tr>
<td><em>/dismount</em> or <em>/d</em></td>
<td>Deprecated. Similar to /unmount or /u.</td>
</tr>
<tr>
<td><em>/unmount</em> or <em>/u</em></td>
<td>Unmount volume specified by drive letter (e.g., /u x). When no drive letter is specified, unmounts all currently mounted VeraCrypt volumes.</td>
</tr>
```

Reply:
Thanks for reminding me. It was on my list, but I forgot to do that. Will get to it soon.

Reply:
Done.